include/tilck/kernel/sys_types.h

```diff
@@ -69,6 +69,10 @@ struct k_utimbuf {
    long modtime;
 };
 
+#define RUSAGE_SELF     0
```

Comment:
At least for the moment, sys_types.h is supposed to just include the system headers and get all the defines from there. The structs and defines in this file are only for the special cases when we cannot just include the system files or when that causes problems because of kernel/userspace incompatibilities. In the future, we might change everything and remove all the includes of system headers. In that case, we'll place everything here. But for the moment, that's the convention.

Reply:
Okay. I didn't notice that it included the system headers. I'll remove the defines and include `sys/resource.h`. Now I understand the meaning of the `k_` prefix